src/package.xml

```diff
@@ -3738,6 +3738,7 @@
         <members>Level__c.All</members>
         <members>Opportunity.To_Be_Acknowledged</members>
         <members>Trigger_Handler__c.All</members>
+        <members>Opportunity_Stage_To_State_Mapping__mdt.All</members>
```

Comment:
It's too bad this doesn't overwrite the default "All" list, but I can't think of a better name 🤷‍♂️  "Stages and States"?

Reply:
I considered All Stage to State Mappings.  If we like that better, so it does not duplicate it, I can change it to that.

Reply:
I do think it alpha sorts them though, and goes with the first one if none are selected, so changing it could cause it to not be selected by default.  Right now, the custom All view is selected.

Reply:
Good point, let's just leave it like this then.